code/modules/uplink/uplink_items.dm

```diff
@@ -918,6 +926,42 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	purchasable_from = ~UPLINK_CLOWN_OPS
 	illegal_tech = FALSE
 
+/datum/uplink_item/ammo/c38
+	name = ".38-special Speed Loader"
+	desc = "A standard issue .38-special speed loader, for use with the Detective's revolver or 'Infiltrator' coat pistol."
+	item = /obj/item/ammo_box/c38
+	cost = 1
+	purchasable_from = ~UPLINK_CLOWN_OPS
+	illegal_tech = FALSE
+
+/datum/uplink_item/ammo/c38blister
+	name = ".38-special 'Blister' Speed Loader"
+	desc = "For when you can't deside between a coat pistol and a dart pistol! These 6 cartridges can \
+			be injected with up to 10 units of your favorite poison for remote application via sidearm."
+	item = /obj/item/ammo_box/c38/dart
+	cost = 2
+	purchasable_from = ~UPLINK_CLOWN_OPS
+	illegal_tech = FALSE
+
+/datum/uplink_item/ammo/c38dumdum
+	name = ".38-special DumDum Speed Loader"
+	desc = "6 specialized fragmenting .38-special catridges, excellent for dispatching unarmored targets. \
+			Shrapnel can embed within the victim and provide a debilitating effect. Not advised for use \
+			against armored targets."
+	item = /obj/item/ammo_box/c38/dumdum
+	cost = 2
+	purchasable_from = ~UPLINK_CLOWN_OPS
+	illegal_tech = FALSE
```

Comment:
These all have big enough variation and different enough roles that it would probably be fine to make them all 1 TC, keeping this gun as the super cheap but kinda crap gun you can buy. All of these ammo types can be printed relatively easilly with the speedloader included unlike other guns which require you to purchase the mags and you can only print the bullets. I think having the more special ammos be cheaper would be good for giving this gun a role of a defensive tool that you can use to slow people down who are onto you, but isn't that good for actual combat

Reply:
All ammos reduced to one TC, weak EMP rounds added (BLK_OUT); Dumdum remain as a cargo only purchase (IIRC the PR for embedding that added them didn't make them printable at all) asides from the uplink.

Blister and BLK_OUT remain uplink exclusive.